### ISSUE TITLE
Get FK information by the table in information_schema.

### ISSUE THREAD

**Issue by ThreeBearsDan** [pingcap/tidb#2697]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
I wanna get the information of foreign key by `information_schema`'s table which store the information of FK. But the table about FK in `information_schema` is empty. Then other way to get FK information is by command of `show create table`. U know this is very hard to parse in this way. There is no comma between two FK when using command `show create table`. A bug?

2. What did you expect to see?
So I hope that Tidb can support that store the FK information in `information_schema`. Thx very much.

3. What did you see instead?
The table about FK in  `information_schema` is empty.

**Comment by shenli** [comment]
@ThreeBearsDan Thanks for your feedback! We will fix this.
